src/main/java/ca/bc/hlth/mohorganizations/CustomExceptionHandler.java

```diff
@@ -0,0 +1,28 @@
+package ca.bc.hlth.mohorganizations;
+
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+import java.util.Arrays;
+
+
+@ControllerAdvice
+public class CustomExceptionHandler extends ResponseEntityExceptionHandler {
+
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(LoggingFilter.class);
+
+    @ExceptionHandler(Exception.class)
+    public final ResponseEntity<String> handleAllExceptions(Exception exception, WebRequest request) {
+
+        LOGGER.error("An error occurred: {}", Arrays.toString(exception.getStackTrace()));
```

Comment:
If you do LOGGER.error(exception.getLocalizedMessage(), exception) (or something to that effect) you will get a nicely printed message and stack trace.